### PR TITLE
Quick Param Updates

### PR DESCRIPTION
**Pull request by mpayson**
For #628 & #626

### CODE REVIEW

File: packages/arcgis-rest-geocoding/test/geocode.test.ts [hunk 1]
```diff
@@ -67,7 +67,13 @@ describe("geocode", () => {
...
     fetchMock.once("*", FindAddressCandidates);
 
-    geocode({ params: { singleLine: "LAX", countryCode: "USA" } })
+    geocode({
+      params: {
+        singleLine: "LAX",
+        countryCode: "USA",
+        outFields: ["Addr_type", "Score"]
+      }
+    })
```
Comment (jgravois) [inline packages/arcgis-rest-geocoding/test/geocode.test.ts]: the params object is a catch all that allows anyone to pass _any_ parameter they want. for a test like this its more helpful to exercise the curated list and confirm we are indeed also passing through 'first class citizen' parameters appropriately.

```ts
geocode({
  singleLine: "LAX",
  countryCode: "USA",
  outFields: ["Addr_type", "Score"]
})
```

File: packages/arcgis-rest-geocoding/test/geocode.test.ts [hunk 2]
```diff
@@ -78,6 +84,10 @@ describe("geocode", () => {
         expect(options.body).toContain("f=json");
         expect(options.body).toContain("singleLine=LAX");
         expect(options.body).toContain("countryCode=USA");
+        const outFieldsEncoded = encodeURIComponent(
+          ["Addr_type", "Score"].join(",")
+        );
+        expect(options.body).toContain(`outFields=${outFieldsEncoded}`);
```
Comment (jgravois) [inline packages/arcgis-rest-geocoding/test/geocode.test.ts]: very nice. 🍨

File: packages/arcgis-rest-geocoding/test/geocode.test.ts (hunk 1 above)
Comment (mpayson) [reply packages/arcgis-rest-geocoding/test/geocode.test.ts]: added [another test](https://github.com/Esri/arcgis-rest-js/pull/641/files#diff-8c669ec214b9561ed12328f5d0518a98R67) for named parameters

**Comment by tomwayson** [comment]
This is in [v2.7.0]( https://github.com/Esri/arcgis-rest-js/releases/tag/v2.7.0)

**Pull request merged**